pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
+name = "cookiecutter-python-template"
+authors = [
+    {name = "Reinout van Rees", email = "reinout@vanrees.org"},
+]
+description = "Cookiecutter template for a plain python project or library"
+readme = "README.rst"
+license = {text = "MIT"}
+version = "0.5.dev0"
+classifiers = [
+        "Development Status :: 4 - Beta",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "License :: OSI Approved :: BSD License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Topic :: Software Development",
+    ]
+dependencies = [
+    "cookiecutter"
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest"
+]
+release = [
+    "zest.releaser >= 8.0.0a1"
+]
```

Comment:
A 'release' extra isn't really standard. And it isn't documented how you can use it => you can better remove it.